src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.CallSiteContainerRewriter.cs

```diff
@@ -61,9 +61,13 @@ public override SyntaxNode VisitLocalDeclarationStatement(LocalDeclarationStatem
                 {
                     node = (LocalDeclarationStatementSyntax)base.VisitLocalDeclarationStatement(node);
 
+                    if (node.Declaration.IsDeconstructionDeclaration)
+                    {// PROTOTYPE(tuples)
```

Comment:
@heejaechang Let me know if you could review this PR and in particular the ExtractMethod refactoring question above. 
I'm happy to stop by and provide some context.